lte/gateway/python/magma/mobilityd/requirements.in

```diff
@@ -0,0 +1,18 @@
+# requirements.in
```

Comment:
How was this curated? Does this replace the dependencies specified in setup.py?

Reply:
It is created by hand during the integration of Bazel-based build for mobilityd. It contains the minimal set of dependencies required by the mobilityd service. In a late step, it could replace setup.py if it includes the dependencies of all python services.

Reply:
understood!